src/mongocxx/validation_criteria.hpp

```diff
@@ -139,17 +139,15 @@ class MONGOCXX_API validation_criteria {
     stdx::optional<bsoncxx::document::view_or_value> _rule;
     stdx::optional<validation_level> _level;
     stdx::optional<validation_action> _action;
+
+    bsoncxx::document::value create_document() const;
```

Comment:
Ditto re ordering, `_`, and making a public inline named `no_deprecated` instead.